src/plugins/caspeco.js

```diff
@@ -0,0 +1,37 @@
+/** @type {import('eslint').Linter.RuleEntry<[]>} */
+const caspeco = {
+	rules: {
+		"discourage-chakra-import": {
+			meta: {
+				type: "suggestion",
+				docs: {
+					description: "Discourage importing anything from @chakra-ui/*",
```

Comment:
Vill man hänvisa till den föreslagna lösningen här i description också kanske?
```suggestion
					description: "Discourage importing anything from @chakra-ui/*. Use imports from "@caspeco/casper-ui-library instead.",
```

Reply:
Det tycker jag!